config/routes.rb

```diff
@@ -266,12 +266,7 @@
 
     root 'families#home'
 
-    resources :family_members do
-      get :resident_index, on: :collection
-      get :new_resident_dependent, on: :collection
-      get :edit_resident_dependent, on: :member
-      get :show_resident_dependent, on: :member
-    end
+    resources :family_members, only: [:new, :create, :show, :index, :edit, :update, :destroy]
```

Comment:
**Nitpick**: Since you’re including all the standard RESTful actions (`:new`, `:create`, `:show`, `:index`, `:edit`, `:update`, `:destroy`), you can simplify the line by removing the `only:` option. Rails will automatically create routes for all seven default actions when `only:` is not specified.

```
resources :family_members

Reply:
Good Catch! Just pushed  the fix.